src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected int status;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.status = 0;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "Y" : "N"); //return Y for Yes or N for No
```

Comment:
Redundant comment.

To further improve, perhaps refactor the "Y" and "N" into a enum or a constant, so that you wouldn't even require a comment to know that it is only a displayed value.